scp/tests/units/test_kb_agents.cpp

```diff
@@ -73,15 +73,15 @@ TEST_F(kbAgentsTest, Test_sc_agent1_scp)
 {
   ScAgentContext & context = *m_ctx;
   context.BeginEventsBlocking();
-  loader.loadScsFile(context, TEST_AGENT_FILES_DIR_PATH + "agent1_proc.scs");
-  loader.loadScsFile(context, TEST_AGENT_FILES_DIR_PATH + "proc_find_all_input.scs");
-  loader.loadScsFile(context, TEST_AGENT_FILES_DIR_PATH + "sc_agent1.scs");
+  loader.loadScsFile(context, TEST_AGENT_FILES_DIR_PATH + "agent_program_search_all_outgoing_arcs_with_relations.scs");
```

Comment:
![image](https://github.com/user-attachments/assets/9446e18b-b07f-4ace-9eda-6e52721f5a6c)
test fails

Reply:
Tests do not work macOS